test/docker/base/Dockerfile

```diff
@@ -79,11 +81,15 @@ RUN mkdir /etc/galaxy && cd /tmp/ansible && mkdir roles && \
     apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 RUN cd $GALAXY_ROOT && \
-    ./scripts/common_startup.sh || { echo "common_startup.sh failed"; exit 1; } && \
-    dev_requirements=./lib/galaxy/dependencies/dev-requirements.txt && \
-    [ -f $dev_requirements ] && $GALAXY_VIRTUAL_ENV/bin/pip install -r $dev_requirements
-
-RUN . $GALAXY_VIRTUAL_ENV/bin/activate && \
+    for VENV in $GALAXY_VIRTUAL_ENV_2 $GALAXY_VIRTUAL_ENV_3; do \
+        export GALAXY_VIRTUAL_ENV=$VENV && \
+        ./scripts/common_startup.sh || { echo "common_startup.sh failed"; exit 1; } && \
+        dev_requirements=./lib/galaxy/dependencies/dev-requirements.txt && \
+        [ -f $dev_requirements ] && $VENV/bin/pip install -r $dev_requirements; done
+
+RUN for VENV in $GALAXY_VIRTUAL_ENV_2 $GALAXY_VIRTUAL_ENV_3; do \
+    export GALAXY_VIRTUAL_ENV=$VENV && \
+    . $GALAXY_VIRTUAL_ENV/bin/activate && \
     pip install psycopg2 && \
     pip install mysql && \
```

Comment:
The loop runs over the rest of this twice but only pre-populates the databases once right? This is fine I'm just double checking - it'd be a little cleaner to end the loop sooner if that is the case but we can merge this as is and do that in a subsequent revision - I know it takes a while to revise these things.

Reply:
Good catch, it does call the script twice, I'll change that